beacon-chain/core/blocks/block_operations.go

```diff
@@ -440,6 +440,46 @@ func ProcessBlockAttestations(
 	return beaconState, nil
 }
 
+func verifyAttestationSig(beaconState *pb.BeaconState, att *pb.Attestation) (bool, error) {
+	attesterIndices, err := helpers.AttestationParticipants(beaconState, att.Data, att.AggregationBitfield)
+	if err != nil {
+		return false, fmt.Errorf("could not get custody bit 1 attestation participants: %v", err)
+	}
+
+	attestorPubKeys := make([]*bls.PublicKey, len(attesterIndices))
+	for idx, participantIndex := range attesterIndices {
+		pubkey, err := bls.PublicKeyFromBytes(beaconState.ValidatorRegistry[participantIndex].Pubkey)
+		if err != nil {
+			return false, fmt.Errorf("could not deserialize proposer public key: %v", err)
+		}
+
+		attestorPubKeys[idx] = pubkey
+	}
+
+	pubkey := attestorPubKeys[0]
+
+	attestationDataHash, err := hashutil.HashProto(&pb.AttestationDataAndCustodyBit{
+		Data:       att.Data,
+		CustodyBit: true,
+	})
+	if err != nil {
+		return false, fmt.Errorf("could not hash attestation data: %v", err)
+	}
+
+	sig, err := bls.SignatureFromBytes(att.AggregateSignature)
+	currentEpoch := helpers.CurrentEpoch(beaconState)
+
+	domain := forkutils.DomainVersion(beaconState.Fork, currentEpoch, params.BeaconConfig().DomainAttestation)
+
+	log.WithFields(logrus.Fields{
+		"pubkey":       fmt.Sprintf("%#x", pubkey.Marshal()),
```

Comment:
This might get weird when 2 attesters are assigned to the same slot and only one attesters pub key gets displayed, I think we can display all the public keys with verbosity as debug